zetaclient/zetacore/client_test.go

```diff
@@ -21,6 +20,7 @@ import (
 	keyinterfaces "github.com/zeta-chain/node/zetaclient/keys/interfaces"
 	"go.nhat.io/grpcmock"
 	"go.nhat.io/grpcmock/planner"
+	"go.uber.org/mock/gomock"
```

Comment:
curious cant we just use the standard library ? 
This repo does look widely used , so I think its fine